src/components/app/admin/Bands/index.tsx

```diff
@@ -58,6 +59,21 @@ function AppBands() {
         });
     }
   };
+  const handleDelete = () => {
+
+
+    BandServ.remove(band)
+      .then((res) => {
+        toast('Band deleted successfully');
+        getBands();
```

Comment:
This should be done automatically. by feathers. I think 🤔

Please test so we don't. replicate. on other models

Thanks for the work. so far